hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/flow/dispatch/child/modules/ChildDispatchModule.java

```diff
@@ -130,6 +131,15 @@ static ServiceApiFactory provideServiceApiFactory(
         return new ServiceApiFactory(stack, configuration, storeMetricsService);
     }
 
+    @Provides
+    @ChildDispatchScope
+    static StoreFactoryImpl storeFactory(
```

Comment:
We could probably return `StoreFactory` here.

Reply:
Fixed, thanks @Neeharika-Sompalli !